Build/FFmpegConfig.sh

```diff
@@ -78,24 +78,33 @@ fi
 if [ "$variant" == "Desktop" ]; then
     configureArgs="\
         $configureArgs \
-        --enable-libx264 \
-        --enable-libx265 \
-        --enable-libvpx \
-        --enable-gpl \
-        --enable-static \
-        --extra-ldflags='-APPCONTAINER:NO -MACHINE:$platform'
+        --disable-encoders \
+        --disable-devices \
+        --enable-hwaccels \
+        --enable-d3d11va \
+        --disable-dxva2 \
+        --disable-programs \
+        --extra-ldflags='-APPCONTAINER:NO -MACHINE:$platform Ws2_32.lib Advapi32.lib User32.lib '
```

Comment:
Hi @softworkz ,

Coming back to this after a few weeks of hindsight, I see you modified some libs to link against, especially under32.lib. 
I don't think we got into too many details, since we focused on the UWP build. Why exactly the current desktop build of ffmpeg wouldn't work that triggered you to make these changes?

is winui somewhat different than an average desktop app(winforms or WPF i.e.) when it comes to runtime?

The way I digested this is that we need to link against different libraries for the desktop/winUI build than UWP, and that UWP kind of lost the "U" part completely.

Reply:
> is winui somewhat different than an average desktop app(winforms or WPF i.e.) when it comes to runtime?

Different in many ways, but always using the system runtime.

I have started a post which gives some overview on the subject: https://github.com/ffmpeginteropx/FFmpegInteropX/issues/390